CHANGES/8104.feature

```diff
@@ -0,0 +1 @@
+Added support for creating, removing, and copying tags for push repositories.
```

Comment:
how about -Added support for tagging and untagging manifests for push repositories